examples/plot_clip.py

```diff
@@ -0,0 +1,156 @@
+"""
+Clip Vector Data with GeoPandas
+==================================================================
+
+Learn how to clip point, line, or polygon geometries to the boundary
+of a polygon geometry using GeoPandas.
+
+"""
+
+###############################################################################
+# Clip Vector Data in Python Using GeoPandas
+# ---------------------------------------------
+#
+# .. note::
+#    The example below will show you how to use the clip() function to clip
+#    vector data such as points, lines and polygons to a vector boundary.
+#
+# The example below walks you through a typical workflow for clipping one
+# vector data file to the shape of another. Both vector data files must be
+# opened with GeoPandas as GeoDataFrames and be in the same Coordinate
+# Reference System (CRS) for the ``clip_shp()`` function in EarthPy to work.
```

Comment:
```suggestion
# Reference System (CRS) for ``clip()`` to work.
```

Reply:
@nkorinek i'm pretty sure they wanted the clip_shp function to be renamed to clip()